assets/src/edit-story/components/canvas/useCanvasSelectionCopyPaste.js

```diff
@@ -190,35 +183,21 @@ function useCanvasSelectionCopyPaste(container) {
         /**
          * Loop through all items in clipboard to check if correct type. Ignore text here.
          */
+        let files = [];
         for (let i = 0; i < items.length; i++) {
-          const item = items[i];
-          if (isValidType(item)) {
-            try {
-              uploadFile(item.getAsFile());
-            } catch (e) {
-              if (!e.isUserError) {
-                e.message = __(
-                  'Sorry, file has failed to upload',
-                  'web-stories'
-                );
-              }
-              showSnackbar({
-                message: e.message,
-              });
-            }
+          const file = items[i].getAsFile();
```

Comment:
It looks like this is not working with videos, the video is not uploaded nor added as an element.
Another note:
It looks like in case of videos, `clipboardData.files` is empty even if a video was actually copied -- meaning that a text element is created with video's name since the `!clipboardData.files?.length` rule above does not prevent processing it.

Does it work for you with videos?

Reply:
See https://github.com/google/web-stories-wp/issues/761#issuecomment-618998121 and https://xwp.slack.com/archives/CCCAUAH6F/p1587725628492100